src/style.css

```diff
@@ -1,3 +1,70 @@
+/* stylelint-disable no-descending-specificity */
+/* stylelint-disable color-hex-case */
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+ul {
+  list-style: none;
+  padding-left: 0;
+  margin-top: 0;
+  margin-bottom: 0;
+}
```

Comment:
In this task, we do not need to reset the styles for all `ul`, so we can move it to `nav__list`